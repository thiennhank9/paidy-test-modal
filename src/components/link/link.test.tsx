import TestRenderer from "react-test-renderer";
import Link from "./link";

describe("Link", () => {
  test("render Link", () => {
    const testRenderer = TestRenderer.create(<Link />);
    expect(testRenderer.toJSON()).toMatchSnapshot();
  });
});