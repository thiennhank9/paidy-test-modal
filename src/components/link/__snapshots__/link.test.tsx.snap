// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link render Link 1`] = `<a />`;
